app/src/main/AndroidManifest.xml

```diff
@@ -32,6 +32,10 @@
             </intent-filter>
         </activity>
 
+        <meta-data
```

Comment:
What's this? It's really necessary?

Reply:
It was added when I import roboto font on Android Studio